modules/oadp-monitoring.adoc

```diff
@@ -0,0 +1,378 @@
+|Total number of attempted backup deletions
+|Counter
+
+|velero_backup_deletion_failure_total
+|Total number of failed backup deletions
+|Counter
+
+|velero_backup_deletion_success_total
+|Total number of successful backup deletions
+|Counter
+
+|velero_backup_duration_seconds
+|Time taken to complete backup, in seconds
+|Histogram
+
+|velero_backup_failure_total
+|Total number of failed backups
+|Counter
+
+|velero_backup_items_errors
+|Total number of errors encountered during backup
+|Gauge
+
+|velero_backup_items_total
+|Total number of items backed up
+|Gauge
+
+|velero_backup_last_status
+|Last status of the backup. A value of 1 is success, 0
+|Gauge
+
+|velero_backup_last_successful_timestamp
+|Last time a backup ran successfully, Unix timestamp in seconds
+|Gauge
+
+|velero_backup_partial_failure_total
+|Total number of partially failed backups
+|Counter
+
+|velero_backup_success_total
+|Total number of successful backups
+|Counter
+
+|velero_backup_tarball_size_bytes
+|Size, in bytes, of a backup
+|Gauge
+
+|velero_backup_total
+|Current number of existent backups
+|Gauge
+
+|velero_backup_validation_failure_total
+|Total number of validation failed backups
+|Counter
+
+|velero_backup_warning_total
+|Total number of warned backups
+|Counter
+
+|velero_csi_snapshot_attempt_total
+|Total number of CSI attempted volume snapshots
+|Counter
+
+|velero_csi_snapshot_failure_total
+|Total number of CSI failed volume snapshots
+|Counter
+
+|velero_csi_snapshot_success_total
+|Total number of CSI successful volume snapshots
+|Counter
+
+|velero_restore_attempt_total
+|Total number of attempted restores
+|Counter
+
+|velero_restore_failed_total
+|Total number of failed restores
+|Counter
+
+|velero_restore_partial_failure_total
+|Total number of partially failed restores
+|Counter
+
+|velero_restore_success_total
+|Total number of successful restores
+|Counter
+
+|velero_restore_total
+|Current number of existent restores
+|Gauge
+
+|velero_restore_validation_failed_total
+|Total number of failed restores failing validations
+|Counter
+
+|velero_volume_snapshot_attempt_total
+|Total number of attempted volume snapshots
+|Counter
+
+|velero_volume_snapshot_failure_total
+|Total number of failed volume snapshots
+|Counter
+
+|velero_volume_snapshot_success_total
+|Total number of successful volume snapshots
+|Counter
+
+|===
+
+[id="Viewing-metrics-OpenShift-Observe-UI_{context}"]
+==  Viewing metrics using the OpenShift Observe UI
+
+The metrics can be viewed in the OpenShift Administrator or Developer view, which must have access to the `openshift-adp project`.
+
+.Procedure
+
+. Select *Observe* and then select *Metrics*.
+.. In the *Developer* view select *Custom query*, or click on the *Show PromQL* link, type the query and click *Enter*.
+.. In the *Administrator* view type the Expression in the text field and select *Run Queries*.
++
+.OADP metrics query
+
+image::oadp-metrics-query.png[OADP metrics query]
```

Comment:
And speaking of the screenshot here, I noticed it was showing the **Developer** perspective, but with the `+` where it is, it makes it look like it's attached to the **Administrator** perspective step. I'd suggest either moving the screenshot up to the **Developer** step, or just remove the `+` here and let it kind of float after the procedure list is technically finished.